src/components/v2/V2Project/V2ProjectSafeDashboard/SafeTransaction.tsx

```diff
@@ -0,0 +1,133 @@
+import { RightOutlined } from '@ant-design/icons'
+import ExternalLink from 'components/ExternalLink'
+import { ThemeContext } from 'contexts/themeContext'
+import { V2UserContext } from 'contexts/v2/userContext'
+import { CSSProperties, useContext, useMemo, useState } from 'react'
+import { formatHistoricalDate } from 'utils/formatDate'
+import { generateSafeTxUrl } from 'utils/safe'
+
+import { SafeTransactionType } from './V2ProjectSafeDashboard'
+
+type SafeTransactionComponentProps = {
+  transaction: SafeTransactionType
+}
+
+const nonceStyle: CSSProperties = {
+  marginRight: '2rem',
+  width: '1rem',
+}
+
+function TransactionHeader({
+  transaction,
+}: {
+  transaction: SafeTransactionType
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  return (
+    <>
+      <div style={{ display: 'flex' }}>
+        <div style={{ ...nonceStyle, color: colors.text.secondary }}>
+          {transaction.nonce}
+        </div>
+        <div>{transaction?.dataDecoded?.method}</div>
+      </div>
+      <div style={{ color: colors.text.secondary }}>
+        {formatHistoricalDate(new Date(transaction.submissionDate).valueOf())}
+        <RightOutlined style={{ marginLeft: 10 }} />
+      </div>
+    </>
+  )
+}
+
+// The components from `TRANSACTION_METHOD_COMPONENTS_MAP` will have specific drawers when expanded
+const ReconfigureFundingCyclesOfTransaction = ({
+  transaction,
+}: SafeTransactionComponentProps) => {
+  // const [expanded, setExpanded] = useState<boolean>(false)
+
+  const { contracts } = useContext(V2UserContext)
+
+  if (!contracts || !transaction.data) return null
+
+  // const decodedData = contracts['JBController'].interface.parseTransaction({
+  //   data: transaction.data,
+  // }).args
+
+  return (
+    <>
+      <TransactionHeader transaction={transaction} />
+
+      {/* {expanded && (
+        <div style={{ marginTop: '2rem', marginLeft: '4rem' }}>
+          <ReconfigurePreview
+            payoutSplits={[]}
+            reserveSplits={[]}
+            fundingCycleMetadata={decodedData._metadata}
+            fundingCycleData={decodedData._data}
+            fundAccessConstraints={decodedData._fundAccessConstraints}
+          />
+        </div>
+      )} 
+      </div>*/}
+    </>
+  )
+}
+
+const GenericSafeTransaction = ({
+  transaction,
+}: SafeTransactionComponentProps) => {
+  return <TransactionHeader transaction={transaction} />
+}
+
+const TRANSACTION_METHOD_COMPONENTS_MAP: {
+  [k: string]: (props: SafeTransactionComponentProps) => JSX.Element | null
+} = {
+  reconfigureFundingCyclesOf: ReconfigureFundingCyclesOfTransaction,
+}
+
+export function SafeTransaction({
+  transaction,
+}: {
+  transaction: SafeTransactionType
+}) {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+  const [hover, setHover] = useState<boolean>(false)
```

Comment:
**suggestion** shouldn't ever set hover state with JS.

I like what you've done with the styling though. It looks like we already have a class for this anyway 💪 

`.hover-bg-l2`